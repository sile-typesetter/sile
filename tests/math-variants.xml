<sile>
<script src="packages/math" />

Math variants, text, MathML:
<mathml>
  <mrow>
  <mo>∀</mo><mi>n</mi><mo>∈</mo><mi mathvariant="double-struck">N</mi>
  <mo>,</mo> <mo>(</mo> <msup> <mi mathvariant="normal">e</mi>
    <mi>n</mi> </msup> <mo>,</mo> <mo mathvariant="double-struck">sin</mo>
  <mo>(</mo><mi>n</mi><mo>)</mo> <mo>)</mo>
  <mo>∈</mo>
  <msup>
    <mrow>
      <mo>(</mo><mi mathvariant="double-struck">R</mi>
      <mo>⧵</mo><mi mathvariant="double-struck">Q</mi><mo>)</mo>
    </mrow>
    <mn>2</mn>
  </msup>
  </mrow>
</mathml>

Math variants, display, MathML:
<mathml mode="display">
  <mrow>
  <mo>∀</mo><mi>n</mi><mo>∈</mo><mi mathvariant="double-struck">N</mi>
  <mo>,</mo> <mo>(</mo> <msup> <mi mathvariant="normal">e</mi>
    <mi>n</mi> </msup> <mo>,</mo> <mo mathvariant="double-struck">sin</mo>
  <mo>(</mo><mi>n</mi><mo>)</mo> <mo>)</mo>
  <mo>∈</mo>
  <msup>
    <mrow>
      <mo>(</mo><mi mathvariant="double-struck">R</mi>
      <mo>⧵</mo><mi mathvariant="double-struck">Q</mi><mo>)</mo>
    </mrow>
    <mn>2</mn>
  </msup>
  </mrow>
</mathml>

Math variants, text, TeX:
<math>
  \forall n \in \mi[mathvariant=double-struck]{N},
  (\mi[mathvariant=normal]{e}^n, \mo[mathvariant=normal]{sin}(n)) \in
  {(\mi[mathvariant=double-struck]{R} \setminus
  \mi[mathvariant=double-struck]{Q})}^2
</math>

Math variants, text, TeX:
<math mode="display">
  \forall n \in \mi[mathvariant=double-struck]{N},
  (\mi[mathvariant=normal]{e}^n, \mo[mathvariant=normal]{sin}(n)) \in
  {(\mi[mathvariant=double-struck]{R} \setminus
  \mi[mathvariant=double-struck]{Q})}^2
</math>

</sile>
