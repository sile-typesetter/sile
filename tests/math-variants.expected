Set paper size 	595.275597	841.8897729
Begin page
Mx 	49.7638
My 	50.0745
Set font 	Gentium Plus;10;400;;normal;;LTR
T	48 68 87 75 w=21.6113	(Math)
Mx 	74.0473
T	89 68 85 76 68 81 87 86 w=33.6035	(variants)
Mx 	107.6508
T	15 w=2.2900	(,)
Mx 	112.6130
T	87 72 91 87 w=16.5332	(text)
Mx 	129.1462
T	15 w=2.2900	(,)
Mx 	134.1084
T	48 68 87 75 48 47 w=34.4336	(MathML)
Mx 	168.5420
T	29 w=2.2900	(:)
Mx 	173.5042
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2043 w=4.8800	(nil)
Mx 	178.3842
T	2725 w=4.7100	(nil)
Mx 	185.9467
T	2051 w=5.0000	(nil)
Mx 	193.7991
T	1880 w=8.2400	(nil)
Mx 	202.0391
T	13 w=2.2000	(nil)
Mx 	205.9058
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	209.4658
T	70 w=4.4700	(nil)
Mx 	213.9358
My 	46.9745
Set font 	Libertinus Math;8;400;;normal;;LTR
T	2725 w=3.7680	(nil)
Mx 	218.1338
My 	50.0745
Set font 	Libertinus Math;10;400;;normal;;LTR
T	13 w=2.2000	(nil)
Mx 	222.0004
T	2967 2957 2962 w=14.3900	(nil)
Mx 	236.3904
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	239.9504
T	2725 w=4.7100	(nil)
Mx 	244.6604
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	248.2204
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	254.6329
T	2051 w=5.0000	(nil)
Mx 	262.4853
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	266.0453
T	1887 w=6.1800	(nil)
Mx 	274.4774
T	2572 w=5.2700	(nil)
Mx 	281.9995
T	1884 w=7.0200	(nil)
Mx 	289.0195
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	292.5795
My 	46.9745
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	49.7638
My 	62.0745
Set font 	Gentium Plus;10;400;;normal;;LTR
T	48 68 87 75 w=21.6113	(Math)
Mx 	74.0597
T	89 68 85 76 68 81 87 86 w=33.6035	(variants)
Mx 	107.6632
T	15 w=2.2900	(,)
Mx 	112.6379
T	71 76 86 83 79 68 92 w=29.3018	(display)
Mx 	141.9397
T	15 w=2.2900	(,)
Mx 	146.9143
T	48 68 87 75 48 47 w=34.4336	(MathML)
Mx 	181.3479
T	29 w=2.2900	(:)
Mx 	234.9044
My 	83.2279
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2043 w=4.8800	(nil)
Mx 	239.7844
T	2725 w=4.7100	(nil)
Mx 	247.2721
T	2051 w=5.0000	(nil)
Mx 	255.0499
T	1880 w=8.2400	(nil)
Mx 	263.2899
T	13 w=2.2000	(nil)
Mx 	267.1566
Mx 	3.0000
T	3653 a=4.2100	(nil)
Mx 	271.3666
T	70 w=4.4700	(nil)
Mx 	275.8366
My 	79.4779
Set font 	Libertinus Math;8;400;;normal;;LTR
T	2725 w=3.7680	(nil)
Mx 	280.0346
My 	83.2279
Set font 	Libertinus Math;10;400;;normal;;LTR
T	13 w=2.2000	(nil)
Mx 	283.9012
T	2967 2957 2962 w=14.3900	(nil)
Mx 	298.2912
Mx 	3.0000
T	3653 a=4.2100	(nil)
Mx 	302.5012
T	2725 w=4.7100	(nil)
Mx 	307.2112
Mx 	2.9900
T	3654 a=4.2100	(nil)
Mx 	311.4212
Mx 	2.9900
T	3654 a=4.2100	(nil)
Mx 	318.4090
T	2051 w=5.0000	(nil)
Mx 	326.1868
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	329.7468
T	1887 w=6.1800	(nil)
Mx 	338.1490
T	2572 w=5.2700	(nil)
Mx 	345.6412
T	1884 w=7.0200	(nil)
Mx 	352.6612
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	356.2212
My 	79.4779
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	49.7638
My 	104.3100
Set font 	Gentium Plus;10;400;;normal;;LTR
T	48 68 87 75 w=21.6113	(Math)
Mx 	74.0495
T	89 68 85 76 68 81 87 86 w=33.6035	(variants)
Mx 	107.6531
T	15 w=2.2900	(,)
Mx 	112.6175
T	87 72 91 87 w=16.5332	(text)
Mx 	129.1507
T	15 w=2.2900	(,)
Mx 	134.1152
T	55 72 59 w=16.2793	(TeX)
Mx 	150.3945
T	29 w=2.2900	(:)
Mx 	155.3590
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2043 w=4.8800	(nil)
Mx 	160.2390
T	2725 w=4.7100	(nil)
Mx 	167.8071
T	2051 w=5.0000	(nil)
Mx 	175.6652
T	1880 w=8.2400	(nil)
Mx 	183.9052
T	13 w=2.2000	(nil)
Mx 	187.7719
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	191.3319
T	70 w=4.4700	(nil)
Mx 	195.8019
My 	101.2100
Set font 	Libertinus Math;8;400;;normal;;LTR
T	2725 w=3.7680	(nil)
Mx 	199.9999
My 	104.3100
Set font 	Libertinus Math;10;400;;normal;;LTR
T	13 w=2.2000	(nil)
Mx 	203.8665
T	84 74 79 w=12.0300	(nil)
Mx 	215.8965
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	219.4565
T	2725 w=4.7100	(nil)
Mx 	224.1665
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	227.7265
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	234.1447
T	2051 w=5.0000	(nil)
Mx 	242.0028
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	245.5628
T	1887 w=6.1800	(nil)
Mx 	253.9972
T	2572 w=5.2700	(nil)
Mx 	261.5215
T	1884 w=7.0200	(nil)
Mx 	268.5415
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	272.1015
My 	101.2100
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	49.7638
My 	116.3100
Set font 	Gentium Plus;10;400;;normal;;LTR
T	48 68 87 75 w=21.6113	(Math)
Mx 	74.0631
T	89 68 85 76 68 81 87 86 w=33.6035	(variants)
Mx 	107.6667
T	15 w=2.2900	(,)
Mx 	112.6447
T	87 72 91 87 w=16.5332	(text)
Mx 	129.1779
T	15 w=2.2900	(,)
Mx 	134.1560
T	55 72 59 w=16.2793	(TeX)
Mx 	150.4353
T	29 w=2.2900	(:)
Mx 	236.0844
My 	137.3920
Set font 	Libertinus Math;10;400;;normal;;LTR
T	2043 w=4.8800	(nil)
Mx 	240.9644
T	2725 w=4.7100	(nil)
Mx 	248.4521
T	2051 w=5.0000	(nil)
Mx 	256.2299
T	1880 w=8.2400	(nil)
Mx 	264.4699
T	13 w=2.2000	(nil)
Mx 	268.3366
Mx 	3.0000
T	3653 a=4.2100	(nil)
Mx 	272.5466
T	70 w=4.4700	(nil)
Mx 	277.0166
My 	133.6420
Set font 	Libertinus Math;8;400;;normal;;LTR
T	2725 w=3.7680	(nil)
Mx 	281.2146
My 	137.3920
Set font 	Libertinus Math;10;400;;normal;;LTR
T	13 w=2.2000	(nil)
Mx 	285.0812
T	84 74 79 w=12.0300	(nil)
Mx 	297.1112
Mx 	3.0000
T	3653 a=4.2100	(nil)
Mx 	301.3212
T	2725 w=4.7100	(nil)
Mx 	306.0312
Mx 	2.9900
T	3654 a=4.2100	(nil)
Mx 	310.2412
Mx 	2.9900
T	3654 a=4.2100	(nil)
Mx 	317.2290
T	2051 w=5.0000	(nil)
Mx 	325.0068
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	328.5668
T	1887 w=6.1800	(nil)
Mx 	336.9690
T	2572 w=5.2700	(nil)
Mx 	344.4612
T	1884 w=7.0200	(nil)
Mx 	351.4812
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	355.0412
My 	133.6420
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	295.2916
My 	780.6177
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20 w=4.6924	(1)
End page
Finish
